src/components/checkbox/checkbox.js

```diff
@@ -1,29 +1,144 @@
-function listener(event) {
-  const element = event.target;
-  if (element.tagName === 'INPUT' && element.type === 'checkbox' && element.hasAttribute('checked') !== element.checked) {
-    if (element.checked) {
-      element.setAttribute('checked', '');
+import mixin from '../../globals/js/misc/mixin';
+import createComponent from '../../globals/js/mixins/create-component';
+import InitComponentBySearch from '../../globals/js/mixins/init-component-by-search';
+// import default from '../lightbox/lightbox';
+
+const stateChangeTypes = {
+  checked: 'true',
+  unchecked: 'false',
+  mixed: 'mixed',
+}
+
+class Checkbox extends mixin(createComponent, InitComponentBySearch) {
+  /**
+   * Checkbox UI.
+   * @extends CreateComponent
+   * @extends InitComponentBySearch
+   * @param {HTMLElement} element The element working as a copy button UI.
```

Comment:
Did you mean “a checkbox UI” instead of “a button UI”? 😉

Reply:
oops :)